.github/workflows/tuttest.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Set Up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```

Comment:
Can you split this into a separate pull request?

Reply:
Done in separate PR